csrc/codegen.cpp

```diff
@@ -402,6 +402,52 @@ class CudaKernelGenerator : private kir::ConstIrVisitor {
     }
   }
 
+  void generateVectorizedLdSt(Val* in, Val* out, CacheOp cache_op) {
```

Comment:
mechanical change, this is lifted from `void handle(const LoadStoreOp* ldst) final`, so it can be shared with TernaryOp handling.